components/formats-bsd/src/loci/formats/services/EXIFServiceImpl.java

```diff
@@ -59,20 +59,19 @@ public EXIFServiceImpl() {
     checkClassDependency(ImageMetadataReader.class);
     // check for xmpcore.jar
     checkClassDependency(com.adobe.xmp.XMPMeta.class);
-    // check for xercesImpl.jar
-    checkClassDependency(org.apache.xerces.dom.NodeImpl.class);
   }
 
   // -- EXIFService API methods --
 
   @Override
   public void initialize(String file) throws ServiceException, IOException {
+    RandomAccessInputStream jpegFile = new RandomAccessInputStream(file);
```

Comment:
This could be simplified into a [try-with-resources](https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html) statement, removing the final `.close()` call.

Reply:
Actually with #3331, https://web-proxy.openmicroscopy.org/east-ci/job/BIOFORMATS-test-folder/61880/console flagged this leaves open file handles and failed the repository tests as expected. Excluding until this is resolved and relaunching the daily jobs.